src/actions/filterActions.js

```diff
@@ -0,0 +1,120 @@
+import queryString from 'query-string'
+import deepEquals from 'deep-equal'
+
+import * as eventActions from './eventActions'
+
+export const CHANGE_EVENT_FILTER = 'CHANGE_EVENT_FILTER'
+
+/*SHAPE OF FILTERS
+filters = {
+    incidentId: 0,
+    eventTypes: [{id: 0, name: testName}, ...]
+    occurredStart: someDateTime,
+    occurredEnd: someDateTime
+}*/
+
+export const changeEventFilter = (history) => (filter) => {
+    getUrlFromFilter(history, filter)
+    return {
+        type: CHANGE_EVENT_FILTER,
+        filter
+    }
+}
+
+export const addFilter = (history) => (filter, eventType) => {
+    let newFilter = {}
+    let oldFilter = filter
+    if (!eventType && !eventType.id) {
```

Comment:
This will crash if eventType is null or undefined. Maybe you meant || instead of &&?

Reply:
Won't the return be triggered with the "!eventType" if the eventType is null?